README.md

```diff
@@ -1,312 +1,323 @@
-   uri_model=quantized_lstm_torch_paths.output_uri,
-)
-
-seg_result = segmenter_lstm_torch_quantized(sample_text, return_logits=True)
-...
-```
-
-```Python
-# BERT models quantized as Torch JIT format
-quantized_bert_torch_paths = segmentador.optimize.quantize_model(
-    segmenter_bert,
-    model_output_format="torch_jit",
-    verbose=True,
-)
-
-segmenter_bert_torch_quantized = segmentador.optimize.TorchJITBERTSegmenter(
-   uri_model=quantized_bert_torch_paths.output_uri,
-)
-
-seg_result = segmenter_bert_torch_quantized(sample_text, return_logits=True)
-...
-```
-
-#### Noise subsegment removal
-Tokens are classified as one out of 4 available classes: No-op (0), Segment (1), Noise Start (2), and Noise End (3). Tokens between a pair of `Noise Start` (inclusive) and the closest `Noise End` or `Segment` (either exclusive) can be removed during the segmentation, by passing the argument `remove_noise_subsegments=True` to the segmenter model:
-
-```python
-seg_result = segmenter(sample_text, remove_noise_subsegments=True)
-```
-
----
-
-### Experimental results
-Experimental results are available in [Result Analsys notebook](./notebooks/6_result_analysis.ipynb), with models tipically achieving per-class precision and recall higher than 95%, despite the problem being severely imbalanced. This same notebook also showcase some tests varying moving window size, moving window shift size, and Bidirectional LSTM models for comparison.
-
----
-
-### Train data
-TODO.
-
----
-
-### Package tests
-Tests for this package are run using Tox and Pytest.
-
----
-
-### License
-[MIT.](./LICENSE)
-
----
-
-### Citation
-```bibtex
-@inproceedings{
-    paper="",
-    author="",
-    date="",
-}
-```
+[![Tests](https://github.com/ulysses-camara/ulysses-segmenter/actions/workflows/tests.yml/badge.svg)](https://github.com/ulysses-camara/ulysses-segmenter/actions/workflows/tests.yml)
+[![Documentation Status](https://readthedocs.org/projects/ulysses-segmenter/badge/?version=latest)](https://ulysses-segmenter.readthedocs.io/en/latest/?badge=latest)
+
+## Brazilian Legal Text Segmenter
+This project presents a Legal Text Segmenter for Portuguese-Brazilian language.
+
+The segmentation problem is formalized here by a 4-multiclass token-wise classification problem. Each token can be classified as follows:
+
+|Class |Description             |
+| :--- | :---                   |
+|0     |No-op                   |
+|1     |Start of sentence       |
+|2     |Start of noise sequence |
+|3     |End of noise sequence   |
+
+
+In a curated dataset, comprised of ground-truth legal text segments, Ulysses Segmenter achieves higher Precision and Recall for the Class I (Segment) than other segmentation tools, such as NLTK, SpaCy and [LexNLP](https://github.com/LexPredict/lexpredict-lexnlp), with the latter being suitable for segmenting legal texts.
+
+| Segmentation Method 	| Precision | Recall 	|
+|:---	    |:---	    |:---	    |
+| NLTK  	| 12.8557% 	| 19.6128% 	|

```

Comment:
Could you either link NLTK, SpaCy, and LexNLP source directory or documentation in this table (or in line 17), please?

Reply:
Done all the requested changes! :)